commands/github.js

```diff
@@ -15,9 +15,7 @@ const run = async (message, args) => {
         await svc.updateByDiscordId(message.author.id, {
             github: args,
         });
-        message.author.send(
-            `Yay! We have set your github profile to \`${args}\``
-        );
+        message.author.send('Yay! We have set your github profile to' + args);
```

Comment:
A space is required.

```suggestion
        message.author.send('Yay! We have set your github profile to ' + args);
```